### ISSUE TITLE
Initializer file removes the belongs_to_required_by_default config in Rails 5

### ISSUE THREAD

**Issue by littleforest** [theSteveMitchell/after_party#34]
In Rails 5, `belongs_to` is required by default.  However, the initializer file for active record:

```
AfterParty.setup do |config|
  # ==> ORM configuration
  # Load and configure the ORM. Supports :active_record (default) and
  # :mongoid (bson_ext recommended) by default. Other ORMs may be
  # available as additional gems.
  require "after_party/active_record.rb"
end
```

is turning that off, likely due to [initialization order](https://github.com/rails/rails/issues/23589#issuecomment-221036080).  Setting

```
ActiveRecord::Base.belongs_to_required_by_default = true
```

in the `config/application.rb` restores the correct behavior, but it would be nice if the gem accounted for this without the override.

**Comment by sfate** [comment]
@littleforest it was fixed at rails `5.1` version.

Ref: https://github.com/rails/rails/issues/23589#issuecomment-305557647

**Comment by littleforest** [comment]
@sfate thanks for the info!
